Lib/base64.py

```diff
@@ -508,10 +508,7 @@ def b85decode(b):
 
 def encode(input, output):
     """Encode a file; input and output are binary files."""
-    while True:
-        s = input.read(MAXBINSIZE)
-        if not s:
-            break
+    while s := input.read(MAXBINSIZE):
         while len(s) < MAXBINSIZE:
```

Comment:
Could this be reduced further by adding `and (ns := input.read(MAXBINSIZE-len(s)))`?

Reply:
```python
def encode(input, output):
    """Encode a file; input and output are binary files."""
    while s := input.read(MAXBINSIZE):
        while len(s) < MAXBINSIZE and ns := input.read(MAXBINSIZE-len(s)):
            s += ns
        line = binascii.b2a_base64(s)
        output.write(line)
```
Does that look right?

Reply:
yes